TestGyCreditExhaustionRestrict is failing in CI

https://app.circleci.com/pipelines/github/magma/magma/6021/workflows/5e767212-50ca-4411-8203-c9d5ac20653e/jobs/60840

```
Failed
=== RUN   TestGyCreditExhaustionRestrict

Running TestGyCreditExhaustionRestrict...
************************* TestRunner setup
Adding Mock HSS service at 192.168.70.101:9204
Adding Mock PCRF service at 192.168.70.101:9202
Adding Mock OCS service at 192.168.70.101:9201
Adding Pipelined service at 192.168.70.101:8443
Adding Redis service at 192.168.70.101:6380
Adding Directoryd service at 192.168.70.101:8443
************************* Configuring 1 UE(s), PCRF instance: PCRF_REMOTE
Added UE to Simulator, HSS_REMOTE, PCRF_REMOTE, and OCS_REMOTE:
	IMSI: 522234939419995	Key: 242e2aa131ea1e169ba5f3c7f1d6265b	Opc: dcda918fd1fd6d2e33423dc432f029ad	Seq: 6410277688524
Successfully configured UE(s)
************************* Adding PCRF Usage Monitor for UE with IMSI: 522234939419995
************************* Adding a Pass-All static rule: static-pass-all-ocs1
************************* Adding a Pass-All static rule: static-pass-all-ocs2
************************* Adding PCRF Rule for UE with IMSI: 522234939419995 with ruleNames=[static-pass-all-ocs1 static-pass-all-ocs2], baseNames=[]
************************* Adding a static rule: restrict-deny-all
************************* Adding a static rule: restrict-pass-user
************************* Authenticating UE with IMSI: 522234939419995
Finished Authenticating UE
************************* Generating Traffic for UE with Req: imsi:"522234939419995" volume:<value:"5M" > bitrate:<value:"60M" > timeout:60 
Record sid:"IMSI659524938666375" rule_id:"redirect" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"restrict-deny-all" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs1" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs2" bytes_tx:4322935 bytes_rx:85720 
Record sid:"IMSI522234939419995" rule_id:"restrict-pass-user" bytes_tx:918469 bytes_rx:8025 ue_ipv4:"192.168.128.2" 
************************* Generating Traffic for UE with Req: imsi:"522234939419995" volume:<value:"2M" > bitrate:<value:"60M" > timeout:60 
Record sid:"IMSI659524938666375" rule_id:"redirect" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"restrict-deny-all" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs1" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs2" bytes_tx:4322935 bytes_rx:85720 
Record sid:"IMSI522234939419995" rule_id:"restrict-pass-user" bytes_tx:2997553 bytes_rx:86731 ue_ipv4:"192.168.128.2" 
Asserting all Gy expectations were met...
Passed!
************************* Generating Traffic for UE with Req: imsi:"522234939419995" volume:<value:"2M" > bitrate:<value:"60M" > timeout:60 
Record sid:"IMSI659524938666375" rule_id:"redirect" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"restrict-deny-all" ue_ipv4:"192.168.128.2" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs1" 
Record sid:"IMSI522234939419995" rule_id:"static-pass-all-ocs2" bytes_tx:2085998 bytes_rx:77593 
Record sid:"IMSI522234939419995" rule_id:"restrict-pass-user" bytes_tx:2997553 bytes_rx:86731 ue_ipv4:"192.168.128.2" 
--- FAIL: TestGyCreditExhaustionRestrict (52.67s)
    gy_enforcement_test.go:98: 
        	Error Trace:	gy_enforcement_test.go:98
        	            				gy_enforcement_test.go:763
        	Error:      	Should be true
        	Test:       	TestGyCreditExhaustionRestrict
        	Messages:   	policy usage: sid:"IMSI522234939419995" rule_id:"restrict-pass-user" bytes_tx:2997553 bytes_rx:86731 ue_ipv4:"192.168.128.2" 
```

--- a/cwf/gateway/integ_tests/gy_enforcement_test.go
+++ b/cwf/gateway/integ_tests/gy_enforcement_test.go
@@ -742,13 +742,10 @@ func TestGyCreditExhaustionRestrict(t *testing.T) {
 	assert.NoError(t, err)
 	tr.WaitForEnforcementStatsToSync()
 
-	// Check that UE mac flow was not removed and data was passed
-	verifyPolicyUsage(t, tr, ue.GetImsi(), "static-pass-all-ocs2", 0, 5*MegaBytes)
-
 	// Wait for service deactivation
 	time.Sleep(3 * time.Second)
 
-	// we need to generate more traffic to hit restrict rule
+	// we need to generate more traffic and validate it goes through restrict rule
 	req = &cwfprotos.GenTrafficRequest{
 		Imsi:    ue.GetImsi(),
 		Volume:  &wrappers.StringValue{Value: "2M"},

